src/modules/details-from-context.js

```diff
@@ -1,9 +1,10 @@
 async function detailsFromContext({ context }) {
   if (!context || !context.payload || !context.payload.issue) {
-    throw new Error('Issue context is not defined');
+    throw new Error("Issue context is not defined");
   }
 
-  const titleParser = /^publish: (?:getsentry\/)?(?<repo>[^/@]+)(?<path>\/[\w./-]+)?@(?<version>[\w.+-]+)$/;
+  const titleParser =
+    /^publish: (?:getsentry\/)?(?<repo>[^/@]+)(?<path>\/[\w./-]+)?@(?<version>[\w.+-]+)$/;
```

Comment:
formatting change

Reply:
please discard the unrelated formatting changes from your PR so I can review the actual change